README.md

```diff
@@ -1,9 +1,18 @@
 # mataDor
-Ett litet system för att hålla reda på när man kan hämta sin mat! Tanken är att hålla det hela enkelt. En jobbare står i baren och när köket ger dem en tallrik mat och tillhörande lapp, skriver denna upp detta i systemet istället för att springa runt. Sedan när någon hämtas så tar personen bort numret. Med denna enkla implementation så är det lätt att hantera när något går fel! :) 
+
+Ett litet system för att hålla reda på när man kan hämta sin mat! Tanken är att hålla det hela enkelt. En jobbare står i baren och när köket ger dem en tallrik mat och tillhörande lapp, skriver denna upp detta i systemet istället för att springa runt. Sedan när någon hämtas så tar personen bort numret. Med denna enkla implementation så är det lätt att hantera när något går fel! :)
 
 ## Setup (linux && mac)
+
 - make sure you have node >=8.12 and yarn installed. (npm -g install yarn if you have npm).
 - clone the repository.
 - run `yarn setup`.
 - open http://localhost:3000/ in your favorite browser.
 - adding numbers is available at /admin
+
+## Deployment
+
+Note: make sure to set env. variables:
```

Comment:
Maybe worth documenting the default values of the env variables as well?

If you want to make the reader happy, you can go the extra mile and create a table, like this:

| Env-variable | Default Value |
| :----------- | :------------ |
| MATADOR_PORT | `3000`        |
| MATADOR_BACKEND_URL | `"http://localhost:" + MATADOR_PORT` |